.github/workflows/ci.yml

```diff
@@ -2,127 +2,114 @@ name: CI
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "dev" ]
```

Comment:
Should this also include `sp1-new`? Or are we going to be depending on `dev` moving forward?

Reply:
Good point, we should include `sp1-new` and also not run `check-downstream-compiles` unless the base branch is `sp1-new`.